src/main/java/net/snowflake/client/jdbc/cloud/storage/StorageHelper.java

```diff
@@ -0,0 +1,36 @@
+package net.snowflake.client.jdbc.cloud.storage;
+
+import java.io.File;
+import java.io.InputStream;
+import net.snowflake.client.jdbc.FileBackedOutputStream;
+
+class StorageHelper {
+  protected static String getStartUploadLog(
```

Comment:
does this method should be protected or package scoped?

Reply:
Right, changed